sbt-metals/src/main/scala/scala/meta/metals/MetalsPlugin.scala

```diff
@@ -72,8 +72,8 @@ object MetalsPlugin extends AutoPlugin {
     },
     allDependencies ++= {
       if (javaSemanticdbEnabled.value)
-        List(
-          "com.sourcegraph" % "semanticdb-javac" % BuildInfo.javaSemanticdbVersion % Configurations.CompileInternal
+        List(Configurations.CompileInternal, Configurations.TestInternal).map(
+          "com.sourcegraph" % "semanticdb-javac" % BuildInfo.javaSemanticdbVersion % _
```

Comment:
Tried it locally and it's need also to cover Integrations tests.
```suggestion
      val existing = ivyConfigurations.value.map(_.name).toSet
      val configurations = Seq(
        Configurations.CompileInternal,
        Configurations.TestInternal,
        Configurations.IntegrationTestInternal
      ).map(_.name).filter(existing.contains).mkString(",")
       List(
          "com.sourcegraph" % "semanticdb-javac" % BuildInfo.javaSemanticdbVersion % configurations
        )
```

Reply:
Thanks, good point!